layouts/quote.vue

```diff
@@ -0,0 +1,40 @@
+<script setup>
+	import BaseLayout from './base-layout.vue';
+
+	const props = defineProps({
+    	author: String,
+		source: String
+	});
+</script>
+
+<template>
+	<base-layout layout="statement">
+	<figure class="quote">
+		<blockquote><slot /></blockquote>
+		<figcaption v-if="props.author">- {{ props.author }}<cite v-if="props.source">, {{ props.source }}</cite></figcaption>
```

Comment:
Pfffff.... the comma shouldn't be part of the `<cite>`, but taking that out makes the template unreadable.

Reply:
(leave it there)